src/server/octobot_service.rs

```diff
@@ -83,8 +83,11 @@ impl OctobotService {
                     (&Post, "/api/users") => UserAdmin::new(self.config.clone(), Op::Create),
                     (&Delete, "/api/user") => UserAdmin::new(self.config.clone(), Op::Delete),
 
-                    (&Get, "/api/repos") => admin::GetRepos::new(self.config.clone()),
-                    (&Post, "/api/repos") => admin::UpdateRepos::new(self.config.clone()),
+                    (&Get, "/api/repos") => RepoAdmin::new(self.config.clone(), Op::List),
+                    (&Put, "/api/repo") => RepoAdmin::new(self.config.clone(), Op::Update),
+                    (&Post, "/api/repos") => RepoAdmin::new(self.config.clone(), Op::Create),
+                    (&Delete, "/api/repo") => RepoAdmin::new(self.config.clone(), Op::Delete),
```

Comment:
rest bikeshed - feel free to ignore:

I would have expected these routes to be:
```
POST    /api/repos      # Create
GET     /api/repos      # Read
PUT     /api/repos/:id  # Update
DELETE  /api/repos/:id  # Delete
```

Reply:
Yeah, I definitely feel bad about these hacky-not-restful-standard routes, but I didn't want to get into URL parsing quite yet -- much easier to match the whole URL. And `/api/repo` seemed better for updates than `/api/repos` which looks more like a collection thing w/o an id on it. Will revisit!